notebooks/amun-inspection-workflow.ipynb

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
Why are notebooks needed in this repo?

Reply:
They aren't, I see them as examples of usage of the Argo Workflows.

Reply:
I was thinking about whether they should rather be added to the `thoth-station/notebooks` repo, but I find it harder to link to when it comes to the outside contributors.

Reply:
Let's move them to `thoth-station/notebooks` - the repo is meant for notebooks and is automatically cloned to ODH JupyterHub instance. This way anyone can immediately try the notebooks. You can link to notebooks from the README file.